server/tests/src/test/resources/database/mariadb.properties

```diff
@@ -3,15 +3,16 @@ data.column.type=VARBINARY(1000)
 timestamp.column.type=BIGINT
 segment.column.type=BIGINT
 database.mode=CONTAINER
-database.container.name=quay.io/bitnami/mariadb
-database.container.tag=10.4
+database.container.name=mariadb
```

Comment:
We should be using `quay.io` here.

Reply:
Unfortunately there's no an official mariadb image on `quay.io` . 
I tried use `quay.io/bitnami/mariadb` but I'm receiving `Illegal mix of collations (latin1_swedish_ci,IMPLICIT) and (utf8mb4_general_ci,COERCIBLE) for operation '='` even if I change `MARIADB_CHARACTER_SET` and `MARIADB_COLLATE` the error doesn't disappear, the same doesn't happen with official mariadb image from dockerhub

Reply:
Hmm I also ran into that issue when I looked into using MariaDB in https://github.com/infinispan/infinispan/pull/8892

Good to know that it doesn't occur with the bitnami image. 

We have the org https://quay.io/organization/infinispan-test, so we could always push the `mariadb` image here as a workaround.